src/main/groovy/org/scoverage/CoverageChecker.groovy

```diff
@@ -56,38 +56,32 @@ class CoverageChecker {
     }
 
     public void checkLineCoverage(File reportDir, CoverageType coverageType, double minimumRate) throws GradleException {
-        NumberFormat defaultNf = NumberFormat.getInstance(Locale.getDefault())
-        checkLineCoverage(reportDir, coverageType, minimumRate, defaultNf)
-    }
-
-    public void checkLineCoverage(File reportDir, CoverageType coverageType, double minimumRate, NumberFormat nf) throws GradleException {
         logger.info("Checking coverage. Type: {}. Minimum rate: {}", coverageType, minimumRate)
 
         XmlParser parser = new XmlParser()
         parser.setFeature('http://apache.org/xml/features/disallow-doctype-decl', false)
         parser.setFeature('http://apache.org/xml/features/nonvalidating/load-external-dtd', false)
 
-        DecimalFormat df = new DecimalFormat("#.##")
-
         try {
             File reportFile = new File(reportDir, coverageType.fileName)
             Node xml = parser.parse(reportFile)
-            Double coverageValue = nf.parse(xml.attribute(coverageType.paramName) as String).doubleValue()
+            Double coverageValue = (xml.attribute(coverageType.paramName) as String).toDouble()
             Double overallRate = coverageType.normalize(coverageValue)
             def difference = (minimumRate - overallRate)
 
             if (difference > 1e-7) {
-                String is = df.format(overallRate * 100)
-                String needed = df.format(minimumRate * 100)
-                throw new GradleException(errorMsg(is, needed, coverageType))
+                throw new GradleException(errorMsg(overallRate, minimumRate, coverageType))
             }
         } catch (FileNotFoundException fnfe) {
             throw new GradleException(fileNotFoundErrorMsg(coverageType), fnfe)
         }
     }
 
     @VisibleForTesting
-    protected static String errorMsg(String actual, String expected, CoverageType type) {
+    protected static String errorMsg(double overallRate, double minimumRate, CoverageType type) {
+        DecimalFormat df = new DecimalFormat("#.##", DecimalFormatSymbols.getInstance(Locale.ENGLISH))
```

Comment:
Sorry -  perhaps https://github.com/maiflai/scalac-scoverage-plugin/blob/67166cd89737bf7a2fc14d8b7a4dafca4f39046d/domain/src/main/scala/scoverage/domain/DoubleFormat.scala#L18 suggests we should be using Locale.US? 

I tried not to bring scoverage classes into the classpath to avoid clashes with other plugins.

Reply:
I do not mind, changed to Locale.US as it also uses "." as a separator.

Reply:
thanks, published 8.0.2